### ISSUE TITLE
The #success in message doesn't work?

### ISSUE THREAD

**Issue by Alber70g** [appleboy/drone-telegram#34]
I feel like the #success in the message doesn't work. It should be a kind of 'if-statement' but it doesn't seem to work. I've tried the example given in the docs, but I think something is wrong.

This is my config right now:

```yaml
  telegram:
    image: appleboy/drone-telegram
    token: ${DRONE_BUILD_BOT_TOKEN}
    to: ddddddd
    message: |
      {{ #success build.status }}
        build {{ build.number }} succeeded. Good job.
      {{ else }}
        build {{ build.number }} failed. Fix me please.
      {{ /success }}
    when: 
      status: [ success, failure ]
```

The error I get: 

```
Parse error on line 1:
Lexer error
Token: Error{"Unexpected character in expression: '#'"}
```

**Comment by Alber70g** [comment]
Apparently it should be `{{#success build.status}}` instead of `{{ #success build.status }}`.

**Comment by appleboy** [comment]
closed via #35
